src/main/webapp/user_profile_script.js

```diff
@@ -160,8 +192,43 @@ async function displayNeedHelpTasks() {
     }
 }
 
+async function displayAwaitVerificationList() {
+    const queryURL = "/mytasks?keyword=Owner&complete=True";
+    const request = new Request(queryURL, {method: "GET"});
+    const response = await fetch(request);
+    const taskResponse = await response.json();
+    const awaitVerifBody = document.getElementById("await-verif-body");
```

Comment:
Same naming change as above :)

Reply:
Sure. I will modify this in the new commit.